coa-helm/local_values.yaml

```diff
@@ -0,0 +1,28 @@
+# This is a template.
+# Don't change this file; copy it instead, and fill in the values needed for deployment.
+# Obtain these values from someone on the development team.
+ 
+config:
+  resourceGroup: "UNDEFINED"
+  azureServicesAuthConnectionString: "UNDEFINED"
+  applicationInsightsAccountName: "UNDEFINED"
+  cosmosDbAccountName: "UNDEFINED"
+  batchAccountName: "UNDEFINED"
+  batchNodesSubnetId: "UNDEFINED"
+  relayListenerSamResourceId: "UNDEFINED"
+  relayListenerConnectionString: "UNDEFINED"
+  relayListenerEndpoint: "UNDEFINED"
+  relayListenerConnectionName: "UNDEFINED"
+  relayListenerSamUrl: "UNDEFINED"
+
+db:
+  name: cromwell_db
+  port: 3306
+  user: cromwell
+  password: cromwell
```

Comment:
We should leave these undefined as well, even though they won't change. The biggest requirement is that we don't want a password checked into source control.

Reply:
I wondered about this -- I certainly don't mind changing these to "UNDEFINED" or something, but doing that would not actually be different in practice. The name, user, and password would just be the string "UNDEFINED"

I'll do it anyway because I agree it's better for the human reader...

Reply:
Actually, maybe I misunderstood: are you suggesting that I actually leave these values completely empty (thus making it invalid YAML)? 

I wonder if that would be better

Reply:
No, I just mean that we shouldn't check the actual, real database connection information into source control. It's fine to check in a fake placeholder value. You can't log into any relevant database using username and password "UNDEFINED." 😄

Reply:
I was thinking that leaving them blank (leaving the YAML in an invalid state) would guard against the possibility of a person accidentally deploying the default values, mistakenly thinking that "UNDEFINED" was somehow special. But maybe I'm overthinking it.